### ISSUE TITLE
Mobile menu cannot scroll or be closed under certain conditions

### ISSUE THREAD

**Issue by ttran34** [ucsc/ucsc-2022#394]
### Description

Can't close or scroll the main menu on an iOS mobile device.
<img width="590" height="1280" alt="Image" src="https://github.com/user-attachments/assets/e9c29eaf-2697-4a08-8493-c1a1adc18172" />

The hamburger menu icon overlaps the white breadcrumb background.
<img width="369" height="800" alt="Image" src="https://github.com/user-attachments/assets/5b88b2bc-a388-469f-a030-9fcca78890a3" />

### Environment info

- Iphone 15
- iOS 26.5
- Mobile browser Safari

### Please confirm that you have searched existing issues in the repo.

Yes

**Comment by knice** [comment]
@luckyluke007 The mobile menu is pushed down because the column that contains the title is pushing the column that contains the mobile menu down beneath it. You can resolve this by reducing the width of the column that contains the title block and testing.

**Comment by knice** [comment]
Similar to #302
